From 17b99656b349b34ad028e646aa857261a2110072 Mon Sep 17 00:00:00 2001
From: Xu Mao <B34688@freescale.com>
Date: Tue, 27 Aug 2013 14:33:35 +0800
Subject: [PATCH 21/70] ENGR00275977 user couldn't do seek action if he/she
 did FF/FB action

remove cactusplayer on MX6SL_EVK

Signed-off-by: Xu Mao <B34688@freescale.com>
---
 CactusPlayer/Android.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/CactusPlayer/Android.mk b/CactusPlayer/Android.mk
index 05168c7..eeab06e 100755
--- a/CactusPlayer/Android.mk
+++ b/CactusPlayer/Android.mk
@@ -1,4 +1,6 @@
+ifeq ($(HAVE_FSL_IMX_GPU3D),true)
 LOCAL_PATH:= $(call my-dir)
+
 include $(CLEAR_VARS)
 
 LOCAL_MODULE_TAGS := optional
@@ -15,3 +17,5 @@ include $(BUILD_PACKAGE)
 
 # Use the folloing include to make our test apk.
 include $(call all-makefiles-under,$(LOCAL_PATH))
+
+endif
-- 
1.8.0

